src/gflags.cc

```diff
@@ -215,6 +215,11 @@ class FlagValue {
     FV_STRING = 6,
     FV_MAX_INDEX = 6,
   };
+
+  friend ValueType FromFlagValueType(FlagValueType flag_type);
```

Comment:
I was not sure, should I delete this internal enum or not, because now it can be replaced with public enum `FlagValueType `.